src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/LagPortSpec.groovy

```diff
@@ -516,6 +516,71 @@ class LagPortSpec extends HealthCheckSpecification {
         lagPort && northboundV2.deleteLagLogicalPort(sw.dpId, lagPort)
     }
 
+    @Tidy
+    def "Able to create/update LAG port with duplicated port numbers on the #sw.hwSwString switch"() {
+        given: "Switch and two ports"
+        def sw = getTopology().getActiveSwitches().get(0)
+        def testPorts = topology.getAllowedPortsForSwitch(sw).take(2)
```

Comment:
fyi:
in groovy you can get elements like in python:
```
groovy:000> a = [1,2,3,4,5]
===> [1, 2, 3, 4, 5]
groovy:000> a[0..1]
===> [1, 2]
groovy:000> a[0]
===> 1
```

Reply:
yes ofc, but in this case it is error friendly when you use .get or .take. That is why I prefer to use it instead of [0..1]. If our code style requires to use direct work with arrays like [0] or [0..1] I will fix that

Reply:
no specific code style, it is up to you